examples/rust/src/main.rs

```diff
@@ -34,10 +35,12 @@ async fn main() -> anyhow::Result<()> {
         .outbox(OutboxServerConfig::default())
         .build()?;
     let cala = CalaLedger::init(cala_config).await?;
+    let random_number = rand::thread_rng().gen_range(0..1000);
```

Comment:
For this I had the idea that we could perhaps inject a test prefix:


${EXAMPLE_PREFIX}. Don't know if you think it would help - but we can set it to something random in the test as well as default it to something random in the example/src/main.rs if its not set.

Reply:
Env variable idea makes sense. Would we ever want to move to having any sort of arg parsing for these?

Reply:
interesting question. I don't think so - certainly most of the time we would want them to execute with defaults.